lib/galaxy/selenium/navigates_galaxy.py

```diff
@@ -1389,22 +1389,29 @@ def wait_for_sizzle_selector_clickable(self, selector):
 
     @retry_during_transitions
     def click_history_options(self):
-        component = self.components.history_panel.options_button_icon
+        if self.is_beta_history():
+            component = self.components.history_panel.options_button_icon_beta
+        else:
+            component = self.components.history_panel.options_button_icon
         component.wait_for_and_click()
 
     def click_history_option(self, option_label_or_component):
         # Open menu
         self.click_history_options()
 
         if isinstance(option_label_or_component, str):
-            option_label = option_label_or_component
-            # Click labelled option
-            self.wait_for_visible(self.navigation.history_panel.options_menu)
-            menu_item_sizzle_selector = self.navigation.history_panel.options_menu_item(
-                option_label=option_label
-            ).selector
-            menu_selection_element = self.wait_for_sizzle_selector_clickable(menu_item_sizzle_selector)
-            menu_selection_element.click()
+            if self.is_beta_history():
+                component = self.beta_history_element("share or publish")
```

Comment:
this is not right

Reply:
Thanks @jmchilton, seeing now that this not used anymore.